### ISSUE TITLE
"Sometimes the lyrics section isn't found"

### ISSUE THREAD

**Issue by ScientiaEtVeritas** [johnwmillr/LyricsGenius#139]
It seems to be a known bug, at least "Sometimes the lyrics section isn't found" suggests so. But it's a quite annoying one, as it's like 50/50 for me. It may depend on the session created. Once it's working it's working for all requests. Once it's not working it keeps not working.

```python
       div = html.find("div", class_="lyrics")
        if not div:
            return None # Sometimes the lyrics section isn't found
```

**Comment by Allerter** [comment]
I think it's because sometimes BeautifulSoup fetches the "new song page", and in Genus's new theme, the `div` class for lyrics is called `Lyrics__Container-sc-1ynbvzw-2 jgQsqn`.
If you change the code you mentioned to the one below it works when the problem is with the `div` class:
```
        if not div:
            div = html.find("div", class_="Lyrics__Container-sc-1ynbvzw-2 jgQsqn")
```
But this workaround has an issue itself: The lyrics that are saved from the new class have no formatting, and there is no `\n` in the lyrics string. Hopefully, @johnwmillr knows how to solve this.

**Comment by Allerter** [comment]
The issue has been resolved in version 2.0.0
